rust/candid/src/bindings/motoko.rs

```diff
@@ -0,0 +1,236 @@
+use super::candid::is_valid_as_id;
```

Comment:
Nice!

I assume this is implementing the `i(…)` function from https://github.com/dfinity/motoko/blob/master/design/IDL-Motoko.md?

Maybe include a link to that document in the code here.